src/pluto/grey_image.cr

```diff
@@ -0,0 +1,63 @@
+require "./image"
+
+class Pluto::GreyImage < Pluto::Image
```

Comment:
From what I see, the correct name would be `GrayscaleImage`.

https://en.wikipedia.org/wiki/Grayscale

Reply:
I'm good either way :+1: I don't gain any additional context with the `scale` suffix, but it is technically correct (the best kind of correct).